walkeeper/Cargo.toml

```diff
@@ -31,6 +31,8 @@ walkdir = "2"
 serde = { version = "1.0", features = ["derive"] }
 hex = "0.4.3"
 const_format = "0.2.21"
+tokio-postgres = "0.7.5"
```

Comment:
tokio-postgres lives in the same repo as rust-postgres which we've already forked. Is it hard to point it to our forked version for less confusion in the future?